va/va.h

```diff
@@ -4208,11 +4210,17 @@ typedef enum {
 } VADecodeErrorType;
 
 /**
- * Client calls vaQuerySurfaceError with VA_STATUS_ERROR_DECODING_ERROR, server side returns
+ * Client calls vaQuerySurfaceError() with the VAStatus obtained from vaSyncSurface(), server side returns
```

Comment:
Need to clear the commemts here:
1. vaQuerySurfaceError with **VA_STATUS_ERROR_DECODING_ERROR** -> query surface mb error etc.
2. vaQuerySurfaceError with **VA_STATUS_ERROR_RESET** -> query surface reset status etc.
In fact, umd doesn't check this error type, may need to correct it in umd.

Reply:
Because we need to recode both MBError and ResetError in the struct `VASurfaceDecodeMBErrors`, so I changed the description to make this struct can store generic Error information. And this change looks confusing now.

Seems `VASurfaceDecodeMBErrors` is designed to store MBError information. Is it reasonable for us to use this structure to store ResetError information?  Do we need to redesign a struct to store ResetError information or just use this struct for both MBError and ResetError, and change the description as @Jexu `s comment.